modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -1264,29 +1264,71 @@ public void testCacheAffinity() throws Exception {
 
     /** */
     public void testCacheConfigNoOutputFormat() throws Exception {
-        testCacheConfig(null);
+        testCacheConfig(null, 1, 1);
     }
 
     /** */
-    public void testCacheConfigSingleLineOutputFormat() throws Exception {
-        testCacheConfig(SINGLE_LINE);
+    public void testCacheConfigSingleLineOutputFormatSingleNodeSignleCache() throws Exception {
+        testCacheConfigSingleLineOutputFormat(1, 1);
     }
 
     /** */
-    public void testCacheConfigMultiLineOutputFormat() throws Exception {
-        testCacheConfig(MULTI_LINE);
+    public void testCacheConfigSingleLineOutputFormatTwoNodeSignleCache() throws Exception {
+        testCacheConfigSingleLineOutputFormat(2, 1);
     }
 
     /** */
-    private void testCacheConfig(OutputFormat outputFormat) throws Exception {
-        Ignite ignite = startGrid();
+    public void testCacheConfigSingleLineOutputFormatTwoNodeManyCaches() throws Exception {
+        testCacheConfigSingleLineOutputFormat(2, 100);
+    }
+
+    /** */
+    public void testCacheConfigMultiLineOutputFormatSingleNodeSingleCache() throws Exception {
+        testCacheConfigMultiLineOutputFormat(1, 1);
+    }
+
+    /** */
+    public void testCacheConfigMultiLineOutputFormatTwoNodeSingleCache() throws Exception {
+        testCacheConfigMultiLineOutputFormat(2, 1);
+    }
+
+    /** */
+    public void testCacheConfigMultiLineOutputFormatTwoNodeManyCaches() throws Exception {
+        testCacheConfigMultiLineOutputFormat(2, 100);
+    }
+
+    /** */
+    private void testCacheConfigSingleLineOutputFormat(int nodesCnt, int cachesCnt) throws Exception {
+        testCacheConfig(SINGLE_LINE, nodesCnt, cachesCnt);
+    }
+
+    /** */
+    private void testCacheConfigMultiLineOutputFormat(int nodesCnt, int cachesCnt) throws Exception {
+        testCacheConfig(MULTI_LINE, nodesCnt, cachesCnt);
+    }
+
+    /** */
+    private void testCacheConfig(OutputFormat outputFormat, int nodesCnt, int cachesCnt) throws Exception {
+        assert nodesCnt > 0;
```

Comment:
Please, change to assertTrue

Reply:
assertTrue("assertion msg", nodesCnt > 0 && cachesCnt > 0);

Ignite ignite = startGrid(nodesCnt)